bool enginestarted
float32 enginerpm
float32 speedmps
float32 odometerm
int32 fuelrangem
float32 throttlepercentage
float32 brakepercentage
float32 steeringpercentage
float32 steeringtorquenm
bool parkingbrake
bool highbeamsignal
bool lowbeamsignal
bool leftturnsignal
bool rightturnsignal
bool horn
bool wiper
bool disengagestatus
int32 drivingmode#ApollocanbusDrivingMode
int32 errorcode#ApollocanbusErrorCode
int32 gearlocation#ApollocanbusGearPosition
float64 steeringtimestamp
apollo_msgs/ApollocommonHeader apolloheader
int32 chassiserrormask 0
apollo_msgs/ApollocommonVehicleSignal signal
apollo_msgs/ApollocanbusChassisGPS chassisgps
apollo_msgs/ApollocommonEngageAdvice engageadvice
apollo_msgs/ApollocanbusWheelSpeed wheelspeed
apollo_msgs/ApollocanbusSurround surround
apollo_msgs/ApollocanbusLicense license
apollo_msgs/ApollocommonVehicleID vehicleid
int32 batterysocpercentage -1
